src/Dot.jsx

```diff
@@ -4,7 +4,7 @@ import styles from './Styles.jsx';
 const Dot = ({selected, handleClick}) => (
     selected ?
     <styles.selectedDot></styles.selectedDot> :
-    <styles.dotBox onClick={() => handleClick()}><styles.dot></styles.dot></styles.dotBox>
+    <styles.dotBox className='box-for-a-dot' onClick={() => handleClick()}><styles.dot></styles.dot></styles.dotBox>
```

Comment:
I see you are using className in some of your files but aren't using a CSS file, why is that?

Reply:
good point. Habit I guess.